docker_push.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+docker push omarvides/docker-ironman-spec
```

Comment:
Where is this repo `omarvides/docker-ironman-spec`?

Reply:
That's a docker hub repository, actually I was thinking and it would be better if it is stored under an ironman account

Reply:
Yeah, I would like that too. No need to change it immediately though.